src/aof.c

```diff
@@ -925,7 +925,7 @@ void aof_background_fsync(int fd) {
 
 /* Close the fd on the basis of aof_background_fsync. */
 void aof_background_fsync_and_close(int fd) {
-    bioCreateCloseJob(fd, 1);
+    bioCreateCloseJob(fd, 1, 1);
```

Comment:
i realize that we're doing a reclaim on the AOF (only when closing a file, or doing a rewrite),
but we didn't do any reclaim in the incremental fsync (bioCreateFsyncJob).
it feels a bit odd, maybe we should be more decisive and either leave the AOF file out of this campaign, or fully include it.
WDYT?

Reply:
I think we'd better address AOF the same  as RDB, since this PR just intend to solve a specific problem we encountered when we do massive upgrade to our cluster initially, so it only reclaim the cache of RDB. I'll submit another pr for it.

Reply:
I think we'd better address AOF the same  as RDB, since this PR just intend to solve a specific problem we encountered when we do massive upgrade to our cluster initially, so it only reclaim the cache of RDB. I'll submit another pr for it.